webapp/publisher/views.py

```diff
@@ -64,6 +64,48 @@ def get_account_details():
     return flask.render_template("publisher/account-details.html", **context)
 
 
+@account.route("/publisher", methods=["GET"])
+@login_required
+def get_publisher_details():
```

Comment:
this seems to be the same function body as `get_account_details` above (apart from the response. Can we reuse the method? Maybe put it in a separate function that both `get_account_details` and `get_publisher_details` both call? Alternatively the endpoint could be `/details.json` with the `.json` signifying to return a json response instead of the html template?

Reply:
@Lukewh We don't have a precedent for using `.json` for API endpoints, other than `account.json` which seems to only be as a workaround too. I agree about the reusing logic though.